common/org.testeditor.dsl.common.ui/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,7 +1,106 @@
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=disabled
```

Comment:
Unfortunately we need to set project specific settings here as the Xtend editor does not properly support `@SupressWarning` - we use `EditorDocumentChange` which is marked as restricted.